.github/workflows/mvn-verify-check/populate-daily-cache.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+max_retry=3; delay=30; i=1
+if [[ $SCALA_VER == '2.12' ]]; then
+    pom='pom.xml'
+    buildver='noSnapshots'
+elif [[ $SCALA_VER == '2.13' ]]; then
+    pom='scala2.13/pom.xml'
+    buildver='noSnapshotsScala213'
```

Comment:
nit: `noSnapshots` should auto-switch while different scala_version, so we may not need to retarget here. 

https://github.com/NVIDIA/spark-rapids/blob/branch-24.08/dist/pom.xml#L61-L70
https://github.com/NVIDIA/spark-rapids/blob/branch-24.08/scala2.13/dist/pom.xml#L61-L70

please help double confirm, thanks